frontend/src/metabase/visualizations/echarts/cartesian/chart-measurements/index.ts

```diff
@@ -25,6 +25,24 @@ import type {
 
 const roundToHundredth = (value: number) => Math.ceil(value * 100) / 100;
 
+const getValuesToMeasure = (min: number, max: number): number[] => {
+  const valuesToMeasure = [min, max];
+
+  const numCandidatesToAdd = 4;
+  const range = max - min;
+  Array.from({ length: numCandidatesToAdd }).forEach((_, i) => {
```

Comment:
`Array.from({ length: ... })` is a bit non-idiomatic way. I think overall this function can be simplified:

```
if (min === max) {
  return [min, max]
}

const stepsCount = 4
const step = (max - min) / stepsCount
const middleValues = [];

for (let i = 0; i < stepCount; i++) {
  middleValues.push(min + step * i)
}

return [...middleValues, ...range]
```

Reply:
Thank you for this! I altered it slightly, so that it is more correct.